evm_loader/docker-compose-test.yml

```diff
@@ -3,7 +3,7 @@ version: "2.1"
 services:
   solana:
     container_name: solana
-    image: cybercoredev/solana:v1.6.7-resources
+    image: cybercoredev/solana:9e68ded325b8ceb25d346cd6e2656f790ba89033
```

Comment:
Use tagged image here (not commit hash).

Reply:
fixed